src/main/java/org/opentripplanner/routing/edgetype/rentedgetype/RentBikeEdge.java

```diff
@@ -0,0 +1,35 @@
+package org.opentripplanner.routing.edgetype.rentedgetype;
+
+import org.opentripplanner.routing.bike_rental.BikeRentalStation;
+import org.opentripplanner.routing.core.State;
+import org.opentripplanner.routing.vertextype.TemporaryRentVehicleVertex;
+
+public class RentBikeEdge extends RentVehicleEdge {
```

Comment:
komentarz, że to jest krawędź do wypożyczania rowerów albo do zamiany dowolnego INNEGO pojazdu na rower (to jest dość istotne, bo teraz ta logika jest taka, że w tej krawędzi możemy tylko wziąć rower, albo zamienić np hulajnogę na rower, a w krawędzi do zwracania (która jest w tym samym wierzchołku) możemy oddać tylko rower).